integration/v3_grpc_test.go

```diff
@@ -598,9 +598,13 @@ func TestV3StorageQuotaAPI(t *testing.T) {
 
 	key := []byte("abc")
 
+	// wait for server to reboot
+	if _, err := kvc.Range(context.TODO(), &pb.RangeRequest{Key: key}, grpc.FailFast(false)); err != nil {
```

Comment:
for the proxy tests to pass it expects the client to do failfast=false for range; this is suddenly breaking the regular test though, so clearly needs more thought...